R/summary_funs.R

```diff
@@ -406,92 +438,120 @@ get_stat <- function(mu, lppd, y_wobs_test, stat, mu.bs = NULL, lppd.bs = NULL,
           rep(1L, y[i_short]))
       }))
       mu <- rep(mu, y_wobs_test$wobs)
-      if (!is.null(mu.bs)) {
-        mu.bs <- rep(mu.bs, y_wobs_test$wobs)
-        # CAUTION: If `y` is allowed to have `NA`s here, then `n_notna.bs` needs
-        # to be adapted:
-        n_notna.bs <- sum(!is.na(mu.bs))
-      }
-      # CAUTION: If `y` is allowed to have `NA`s here, then `n_notna` needs to
-      # be adapted:
-      n_notna <- sum(!is.na(mu))
```

Comment:
I think these lines need to be re-added, but replacing `n_notna` with `n`. Otherwise, `n` will still refer to the length of the aggregated dataset although the dataset (the Bernoulli trials) were de-aggregated here.

Reply:
I made a change, but not certain if I did understand your comment correctly, so please do check again

Reply:
Thanks, this is ok now.